tests/test_properties.py

```diff
@@ -75,6 +78,82 @@ def test_create_update_delete_property_files(self):
         for full_file_path in all_files_actual_properties:
             self.assertFalse(full_file_path.exists())
 
+    def test_multiline(self):
+        self.edit_update_compare(
+            "customers_multiline.yml", target_model="customers"
+        )
+
+    def test_multiline_missing_column(self):
+        self.edit_update_compare(
+            "customers_multiline_missing_column.yml",
+            target_model="customers",
+            expected_file="customers_multiline.yml",
+        )
+
+    def test_multiline_pipe(self):
+        self.edit_update_compare(
+            "customers_multiline_pipe.yml", target_model="customers"
+        )
+
+    def test_multiline_quotes(self):
+        self.edit_update_compare(
+            "customers_multiline_quotes.yml",
+            target_model="customers",
+            expected_file="customers_multiline_quotes_expected.yml",
+        )
+
+    def test_long_string(self):
+        self.edit_update_compare(
+            "customers_long_string.yml",
+            target_model="customers",
+            expected_file="customers_long_string_expected.yml",
+        )
+
+    def test_comment(self):
+        self.edit_update_compare(
+            "customers_comment.yml", target_model="customers"
+        )
+
+    def edit_update_compare(
+        self, source_file, target_model="customers", expected_file=None
+    ):
+        target_path = Path(
+            self.project_dir, "models", "marts", "core", f"{target_model}.yml"
+        )
+        source_path = Path(
+            self.test_base_dir, "test_property_files", source_file
+        )
+        if expected_file:
+            expected_path = Path(
+                self.test_base_dir, "test_property_files", expected_file
+            )
+        else:
+            expected_path = source_path
+        shutil.copy(source_path, target_path)
+        with patch('builtins.input', return_value='y'):
+            properties.update(
+                self.ctx,
+                select=target_model,
+                project_dir=self.project_dir,
+                profiles_dir=self.profiles_dir,
+                log_level='DEBUG',
+            )
+            # check the content
+        self.logger.info(
+            f"Comparing content of files {target_path} and {expected_path}"
+        )
+        with open(target_path) as f:
+            content = '\n'.join(f.readlines())
+        self.logger.debug(f"Target content is \n{content}")
+        with open(expected_path) as f:
+            content = '\n'.join(f.readlines())
+        self.logger.debug(f"Expected content is \n{content}")
```

Comment:
Depending on what diff information we end up logging in `self.compare_files` I think we should reevaluate if we need to log the full contents of the files here.

Reply:
yes, you're right. These logs are reminiscences of trying to figure out why using `filecmp` did not work as expected and caused my tests to fail ;)

Reply:
fixed in https://github.com/Dashlane/dbt-invoke/pull/20/commits/2c02fa485b99eaf1606df7f2fe82315f8136af6b